src/code/gamealloc.c

```diff
@@ -12,6 +12,7 @@ void GameAlloc_Log(GameAlloc* this) {
     }
 }
 
+#if OOT_DEBUG
```

Comment:
Could you ifdef-out or delete the declarations in `functions.h` for the ones that are disabled here?

Reply:
Probably can delete `func_800C55D0` there too, that seems to be a missed rename of `GameAlloc_Log`